DataCapturing/Source/Capturing/SensorCapturer.swift

```diff
@@ -30,31 +30,11 @@ import os.log
  */
 class SensorCapturer {
 
+    // MARK: - Constants
     /// The log used to identify messages from this class.
     private static let log = OSLog(subsystem: "SensorCapturer", category: "de.cyface")
-    /// Internal storage for the kernel boot time, to avoid regular recalculation
```

Comment:
should we not add some documentation to the code why we're using the system time now (again) instead of using the CPU- and sensor event time?

We probably remember this for a few years but imagine someone else will maintain this code in a few years and there are no team members available who can remember this incident....

Just add the comment you left in the task STAD-258 in the code where you use the system time.